src/test/java/org/takes/rq/multipart/ContentDispositionHeaderText.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
@Serranya I'm sorry, but I didn't understand why we need a class just to represent this header. Could you explain me?

Reply:
I don't _need_ a class to represent this Header. I just wanted to avoid writing Content-Disposition over and over.

If you want me to I can remove those two classes and only use FormattedString to write the header Strings.

Reply:
@Serranya I think (in this case) write `FormattedText` few times it's better than create a new class just to write one HTTP header. So, let's use only `FormattedText` and be happy 😃